components/analytics/format.tsx

```diff
@@ -60,6 +61,27 @@ export function formatPostsPerDayData(data: any) {
     return chartData;
 }
 
+export function syncronizeChartData(...chartDatas: any[]) {
```

Comment:
Perhaps a high-level summary of this function for those not familiar with the data model?
```
// syncronizeChartData converges on a uniform set of labels for all entries in the given chart data. If a given label wasn't already present in the chart data, a 0-valued data point at that label is added.
//
// For date-labelled charts, this ensures that each charts starts and ends on the same interval, even if data for part of that interval was never collected.

Reply:
Great suggestion. Added!